packages/cli/hops.js

```diff
@@ -159,11 +159,14 @@ var localCliPath = getLocalCliPath();
 
 var isInsideHopsProject = false;
 try {
-  var manifest = require(path.resolve(process.cwd(), 'package.json'));
+  var hopsRoot = require('pkg-dir').sync(process.cwd());
+  var manifest = require(path.join(hopsRoot, 'package.json'));
   var dependencies = Object.keys(manifest.dependencies || {}).concat(
     Object.keys(manifest.devDependencies || {})
   );
-  isInsideHopsProject = dependencies.indexOf('hops-local-cli') > -1;
+  isInsideHopsProject = PACKAGES_TO_INSTALL.every(function(dependency) {
```

Comment:
Why, oh why, are we using a constant-ish array (PACKAGES_TO_INSTALL) with just a single element? ;)